test-utils/src/asserts.rs

```diff
@@ -0,0 +1,31 @@
+const DISABLE_STRING_CHECKS_ENV_VAR: &str = "DEFUSE_SKIP_STRING_ERROR_CHECKS";
+
+pub trait ResultAssertsExt {
+    fn assert_error_contains(&self, s: &str);
+}
+
+impl<T, E> ResultAssertsExt for Result<T, E>
+where
+    E: ToString,
+{
+    fn assert_error_contains(&self, to_contain: &str) {
```

Comment:
Make it accept `impl Display`, so you can `.assert_err_contains(MyCustomError::Variant)` and return `E` for better usage
```suggestion
    #[track_caller]
    fn assert_err_contains(&self, to_contain: impl Display) -> E {
```